### ISSUE TITLE
Add a "reject all" batch action

### ISSUE THREAD

**Issue by bugzilla-to-github** [mozilla/pontoon#2082]
*This issue was created automatically by a [script](https://github.com/mathjazz/bugzilla2github/).*
## [Bug 1386267](https://bugzilla.mozilla.org/show_bug.cgi?id=1386267)
Bug Reporter: @adngdb
CC:   @mathjazz, soniasingla.1812@gmail.com
Depends on:   [Bug 1288956](https://bugzilla.mozilla.org/show_bug.cgi?id=1288956)
See also:     https://bugzilla.mozilla.org/show_bug.cgi?id=1267548
See also:     https://bugzilla.mozilla.org/show_bug.cgi?id=1324411

In bug 1288956, we added a batch action to reject suggestions. Matjaz wants to do the following instead:

> I think we should split this into 2 actions:
> 
>    Reject All - which will reject all translations for these strings, including approved
>    Reject Suggestions Only - which will only reject suggestions (the current implementation)
> -- https://github.com/mozilla/pontoon/pull/634/files#r130123868

While we're at it, it's probably a good idea to refactor that code a bit to make adding new actions easier.

**Comment by mathjazz** [comment]
As agreed on Slack, the risks of such feature outweigh the benefits.
